venv.go

```diff
@@ -5,9 +5,11 @@ package main
 import (
```

Comment:
This file shouldn't be a part of this change.

Reply:
it's because I branched off the other PR assuming that it would get merged by the time this pr gets reviewed